src/packages/src/xroad/ubuntu/generic/xroad-securityserver-is.postinst

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+if [ "$1" = configure ]; then
+  invoke-rc.d --quiet xroad-addon-messagelog try-restart || true
```

Comment:
If there are messagelog customizations, one needs to restart xroad-proxy (there is no xroad-addon-messagelog service)

Reply:
Fix'd